benchmark/run_bench.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+
+rs=(1)
+cs=(1)
+w=10
+d=10
+reqs=(1)
+resps=(1)
+rpc_types=(unary)
+
+# idx[0] = idx value for rs
```

Comment:
I don't get it...what is idx?

Reply:
All input parameters that are arrays, need an index to go over them while keeping other parameters' index unchanged. This way we can generate all combinations without using nested for loops